autogpt_platform/backend/backend/blocks/data_manipulation.py

```diff
@@ -18,6 +18,19 @@ class Input(BlockSchema):
             placeholder="e.g., {'name': 'Alice', 'age': 25}",
         )
 
+        @classmethod
+        def get_field_schema(cls, field_name: str) -> dict[str, Any]:
```

Comment:
Where is this used ?

Reply:
its called by the smart decsion maker to build the schema